torch/csrc/jit/script/compiler.cpp

```diff
@@ -1328,6 +1328,37 @@ struct to_ir {
     return n;
   }
 
+  void matchSchemaAndLiftConstantAttributes(
+      const SourceRange& loc,
+      Node* n,
+      std::vector<Value*> input_vals,
```

Comment:
It seems like `input_vals` and `name` are redundant to `n->kind()` and `n->inputs()` is there a reason they are separate arguments?